src/server/helpers/config.js

```diff
@@ -11,6 +11,8 @@ import dotenv from 'dotenv';
 const CLIENT_SIDE_WHITELIST = [
   'NODE_ENV',
   'BASE_URL',
+  'POLLER_REQUEST_BUILDS',
```

Comment:
We need that to override it on the script commandline e.g. `npm run poll-repositories -- --env=environments/dev.env --POLLER_REQUEST_BUILDS`